### DISCUSSION TITLE
Max Depth support?

### DISCUSSION THREAD

**Discussion by himanshusahu06** [nlohmann/json#4139] [General]
Is there a way to set hard limit on nesting (nesting)? For example, parser should error out if depth is more than certain value.

**Comment by himanshusahu06** [comment]
@nlohmann

**Comment by gregmarr** [comment]
You can use a callback to cause it to ignore anything below a specific depth, but I don't think it causes an error.  
https://json.nlohmann.me/features/parsing/parser_callbacks/
